### PR TITLE
Cypress/E2E: Fix email verification that works both with current, Cloud and parallel testing (part 2)

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix email verification that works both with current, Cloud and parallel testing.

Note: Submitted on top of https://github.com/mattermost/mattermost-webapp/pull/7140 for ease of (code) review.

I'll handle the merging in sequence.

### CODE REVIEW

File: e2e/cypress/integration/email/mention_email_notification_spec.js [hunk 1]
```diff
@@ -57,10 +56,9 @@ describe('Email notification', () => {
         const mailUrl = getEmailUrl(baseUrl);
 
         cy.task('getRecentEmail', {username: mentionedUser.username, mailUrl}).then((response) => {
-            const messageSeparator = getEmailMessageSeparator(baseUrl);
-            verifyEmailNotification(response, config.TeamSettings.SiteName, testTeam.display_name, 'Town Square', mentionedUser, testUser, text, config.EmailSettings.FeedbackEmail, config.SupportSettings.SupportEmail, messageSeparator);
+            verifyEmailNotification(response, config.TeamSettings.SiteName, testTeam.display_name, 'Town Square', mentionedUser, testUser, text, config.EmailSettings.FeedbackEmail, config.SupportSettings.SupportEmail);
 
-            const bodyText = response.data.body.text.split('\n');
+            const bodyText = response.data.body.text.split('\n').map((d) => d.trim());
```
Comment (josephbaylon) [inline e2e/cypress/integration/email/mention_email_notification_spec.js:61]: Looks like splitting and creating array of trimmed text is done on many places. Can we pull this out as a utility function?

**Review by josephbaylon — APPROVED** [review]
LGTM Just a minor question.

File: e2e/cypress/integration/email/mention_email_notification_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/email/mention_email_notification_spec.js:61]: Yup, have seen that too.  I'll update.

Comment (saturninoabril) [reply e2e/cypress/integration/email/mention_email_notification_spec.js]: Done, PTAL. Thanks!

**Review by prapti — APPROVED** [review]
LGTM!

**Comment by prapti** [comment]
Holding off on merge now based on your comment above :)

**Pull request merged**